html/clock.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Clock</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/static/css/bootstrap.min.css">
+</head>
+
+<body>
+    {{template "navbar"}}
+
+    <div class="container">
+        {{if ne .ErrorMessage ""}}
+        <div class="alert alert-danger">
+            {{.ErrorMessage}}
+        </div>
+        {{end}}
+
+        {{if ne .Message ""}}
+        <div class="alert alert-success">
+            {{.Message}}
+        </div>
+        {{end}}
+
+        <div class="container pt-5 pl-0">
+            <h2>Clock<br></h2>
+        </div>
+        <hr>
+
+        <form action="/clock" method="POST">
+          <div class="form-row">
+            <div class="form-group col-md-6">
+              <label for="rtcdate">Hardware Clock Date</label>
+              <input type="date" class="form-control" name="rtcdate" value="{{.RTCDate}}" disabled>
+            </div>
+          </div>
+
+          <div class="form-row">
+            <div class="form-group col-md-6">
+              <label for="rtctime">Hardware Clock Time</label>
+              <input type="time" class="form-control" name="rtctime" value="{{.RTCTime}}" disabled>
+            </div>
+          </div>
+
+          <div class="form-row">
+            <div class="form-group col-md-6 pt-4">
+              <label for="systemdate">System Date</label>
+              <input type="date" class="form-control" name="systemdate" value="{{.SystemDate}}" disabled>
+            </div>
+          </div>
+
+          <div class="form-row">
+            <div class="form-group col-md-6 ">
+              <label for="systemtime">System Time</label>
+              <input type="time" class="form-control" name="systemtime" value="{{.SystemTime}}" disabled>
+            </div>
+          </div>
+
+
+            <div class="form-row">
+              <div class="form-group col-md-6">
+                <button id="setButton" type="submit" name="action" onclick="getTime()" class="btn btn-primary">Update Time</button>
+              </div>
+            </div>
+
+            <input id=currenttime type="text" class="form-control" name="currenttime" hidden>
+
+          </form>
+        
+    </div>
+
+    <script src="/static/js/jquery-3.3.1.slim.min.js"></script>
+    <script src="/static/js/bootstrap.min.js"></script>
+
+    <script>
+      $(document).ready(function(){
+          document.getElementById("btnBack").href="/advanced";
+          document.getElementById("navDeviceName").href="/advanced";
+      });
+    </script>
+
+<script>
+  function getTime() {
+    // Populate a hidden field so that the server can access it.
+    var d = new Date();
+    $('#currenttime').val(d.toISOString());
+
+  }
```

Comment:
the contents of this `<script>` might as well be merged with the one just above it.